src/main/java/com/ksm/wstbackoffice/exception/ControllerExceptionHandler.java

```diff
@@ -0,0 +1,23 @@
+package com.ksm.wstbackoffice.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+
+import java.util.Date;
+
+@ControllerAdvice
+public class ControllerExceptionHandler {
```

Comment:
Please put this class into the package controller/handler

Reply:
done